src/sentry/rules/conditions/event_frequency.py

```diff
@@ -426,7 +438,7 @@ def batch_query_hook(
     ) -> dict[int, int]:
         batch_sums: dict[int, int] = defaultdict(int)
         groups = Group.objects.filter(id__in=group_ids).values(
-            "id", "type", "project__organization_id"
+            "id", "type", "project_id", "project__organization_id"
```

Comment:
this (and the one below) adds an extra column to the select query -- necessary to make the calls below safe though (matching TypedDict)